modules/sdk-coin-atom/package.json

```diff
@@ -44,7 +44,6 @@
     "@bitgo/statics": "^14.1.0",
     "@bitgo/utxo-lib": "^7.5.0",
     "@cosmjs/amino": "^0.29.5",
-    "@cosmjs/crypto": "^0.29.5",
```

Comment:
No package lock changes?

Reply:
Isn't the lock-file automatically generated?
I do not see it in other coin modules
https://github.com/BitGo/BitGoJS/tree/master/modules/sdk-coin-atom
https://github.com/BitGo/BitGoJS/tree/master/modules/sdk-coin-sol

Reply:
Yeah, Lock files are autogenerated when you do `yarn`  again if you made changes to dependencies. I checkout locally and seems like it's a peer dependencies for some of the packages like `@cosmjs/amino` etc, probably explains why lock file was not modified.

Reply:
Oh yes - they're dependencys of each other, thanks for checking!